FAQ.md

```diff
@@ -386,13 +386,23 @@ The offline node usually indicates the TiKV node. You can determine whether the
 
 The lease parameter (`--lease=60`) is set from the command line when starting a TiDB server. The value of the lease parameter impacts the Database Schema Changes (DDL) speed of the current session. In the testing environments, you can set the value to 1s for to speed up the testing cycle. But in the production environments, it is recommended to set the value to minutes (for example, 60) to ensure the DDL safety.
 
+#### What is the processing time of a DDL operation?
+
+Generally, the processing time of a DDL statement can be divided into three conditions:
```

Comment:
"divided into" is not appropriate.

Reply:
In this answer, it means we should consider the following three scenarios. The processing time is different for each scenario.